react/legacy/components/ProductSummarySmall.js

```diff
@@ -32,12 +31,17 @@ class ProductSummarySmall extends Component {
       'overflow-hidden br3 h-100 flex flex-column justify-between center tc'
     )
 
-    const summaryClasses = classNames(`${productSummary.element} pointer pt3 pb4 flex flex-column`, {
-      'bb b--muted-4 mh2 mt2': showBorders,
-    })
+    const summaryClasses = classNames(
+      `${productSummary.element} pointer pt3 pb4 flex flex-column`,
```

Comment:
nit: no need for template string when using classNames

Reply:
Wasn't me 😢

Reply:
sorry m8

Reply:
doit